packages/veritone-react-common/src/components/DataPicker/UploaderViewContainer/index.js

```diff
@@ -46,75 +45,77 @@ const UploadViewContainer = ({
     [uploadError]: 'failure'
   }[true];
   const icon = {
-    success: ( <CheckCircle style={{ fill: green[500] }} /> ),
-    failure: ( <Info style={{ color: '#F44336' }} /> ),
-    warning: ( <Warning style={{ color: '#ffc107' }} /> )
+    success: (<CheckCircle style={{ fill: green[500] }} />),
+    failure: (<Info style={{ color: '#F44336' }} />),
+    warning: (<Warning style={{ color: '#ffc107' }} />)
   }[completeStatus];
   const uploadDisabled = (
-      percentByFiles
-      && percentByFiles.length > 0
-      && percentByFiles.some(percent => percent.value.percent > 0)
-    ) || (uploadedFiles && !uploadedFiles.length);
+    percentByFiles
+    && percentByFiles.length > 0
+    && percentByFiles.some(percent => percent.value.percent > 0)
+  ) || (uploadedFiles && !uploadedFiles.length);
 
   return (
     <div className={styles['uploader-container']} style={containerStyle}>
```

Comment:
@SteveShaffer Shall we use kabob case or camel case for classes (just asking for consistency)?

Reply:
@MmtBkn, @SteveShaffer  In my opinion, we should use kabob case for classes in .css files, but for .scss module I am not sure

Reply:
It looks like camelCase is the consistent option.  If the classes were gonna hit the DOM I would say kebab-case, but with classNames, those are just reference names anyway so please go with camelCase.